chainer/link.py

```diff
@@ -357,6 +357,16 @@ def copyparams(self, link):
         for name in self._params:
             dst[name].copydata(src[name])
 
+    def cleargrads(self):
+        """Clears all gradient arrays.
+
+        This method should be called before the backward computation at every
+        iteration of the optimization.
```

Comment:
The note is almost same as that of zerograds, so users may be confused. Would you update the docstring of zerograds? I think new users should use cleargrads in most cases.